docs/user_guide/modal-publishing.md

```diff
@@ -0,0 +1,151 @@
+## 1. Create a Modal Account
+
+**NOTE:** This step requires a free [GitHub](https://github.com) account to authenticate with Modal.
+
+Create a free Modal account: [signup](https://modal.com/signup)
+
+## 2. Author a Modal App
+
+### Create a Modal file
+
+Modal files are regular python scripts. Create a new python file in your project:
+
+```bash
+$ touch my_app.py
+```
+
+### Imports
+
+```python
+# my_app.py
+
+# Import the Modal SDK
+import modal
+```
+
+### Define the Modal App
+
+Create a `modal.App` obeject where your custom functions will be registered.
+Modal apps need to be assigned to a variable named `app` and in the global scope.
+
+```
+app = modal.App("my-cool-app")
+```
+
```

Comment:
I think we need a section on images too so people can specify requirements -- it can be brief but it should link out to the modal docs/playground https://modal.com/playground/custom_container